RNTester/RCTTest/RCTTestRunner.m

```diff
@@ -199,8 +201,10 @@ - (void)runTest:(SEL)test module:(NSString *)moduleName
         configurationBlock(rootView);
       }
 
+      RCTImageLoader *imageLoader = [bridge moduleForClass:[RCTImageLoader class]];
```

Comment:
The Circle CI test `test_ios_unit_frameworks` is failing with a link failure Undefined symbol: `_OBJC_CLASS_$_RCTImageLoader`.   It would appear that either RCTTest.framework needs to have RCTImage added as a dep, or the change here in RCTTestRunner could be refactored to use `NSClassFromString(@"RCTImageLoader")` and use performSelector on activeTasks.